scrapi/harvesters/osf.py

```diff
@@ -96,9 +95,10 @@ def schema(self):
         }
 
     def harvest(self, start_date=None, end_date=None):
-
-        start_date = start_date or date.today() - timedelta(settings.DAYS_BACK)
-        end_date = end_date or date.today()
+        # Always harvest a 2 day period starting 2 days back to honor time given
```

Comment:
I just added that in as that seemed to be the standard in other harvesters - 2 days just to make sure we cover everything in the gaps between harvests